### PR TITLE
NXP-30392: migrate coldstorage infobar code

### PR DESCRIPTION
**Pull request by andreacornaglia**
depends on https://github.com/nuxeo/nuxeo-web-ui/pull/1227

### CODE REVIEW

File: nuxeo-coldstorage-web/elements/nuxeo-coldstorage-infobar.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+          @apply --layout-justified;
+          padding: 8px;
+          margin-bottom: 16px;
+          box-shadow: 0 3px 5px rgba(0, 0, 0, 0.04);
+          background-color: var(--nuxeo-box);
+        }
+        .bar.coldStorage {
+          background: black;
+          color: white;
+        }
```
Comment (Gabez0r) [inline nuxeo-coldstorage-web/elements/nuxeo-coldstorage-infobar.js:46]: These casses are used together, so perhaps they could be merged.

File: nuxeo-coldstorage-web/nuxeo-coldstorage.html [hunk 2]
```diff
@@ -35,3 +35,9 @@
     ></nuxeo-move-contents-to-coldstorage-button>
   </template>
 </nuxeo-slot-content>
+
+<nuxeo-slot-content name="coldStorageDocumentInfoBar" slot="DOCUMENT_INFOBAR">
+  <template>
+    <nuxeo-coldstorage-infobar document="[[document]]"></nuxeo-coldstorage-infobar>
+  </template> </nuxeo-slot-content
```
Comment (Gabez0r) [inline nuxeo-coldstorage-web/nuxeo-coldstorage.html:42]: The formatting seems to be a bit off here.

File: nuxeo-coldstorage-web/elements/nuxeo-coldstorage-infobar.js (hunk 1 above)
Comment (andreacornaglia) [reply nuxeo-coldstorage-web/elements/nuxeo-coldstorage-infobar.js]: thanks! done

File: nuxeo-coldstorage-web/nuxeo-coldstorage.html (hunk 2 above)
Comment (andreacornaglia) [reply nuxeo-coldstorage-web/nuxeo-coldstorage.html]: indeed. fixed

File: nuxeo-coldstorage-web/elements/nuxeo-coldstorage-infobar.js [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+          margin-right: 3px;
+        }
+      </style>
+      <dom-if if="[[_contentStoredInColdStorage(document)]]">
+        <template>
+          <div id="coldStorageInfoBar" class="bar coldStorage">
+            <div class="layout horizontal center flex">
+              <iron-icon noink icon="coldstorage:default"></iron-icon>
+              <span class="storedInColdStorage">[[i18n('documentPage.content.storedInColdStorage')]]</span>
+              <span>[[i18n('documentPage.content.lowResolutionPreview')]]</span>
```
Comment (Gabez0r) [inline nuxeo-coldstorage-web/elements/nuxeo-coldstorage-infobar.js:55]: These labels should be moved as well. Please don't forget to warn Manon about this so she can take care of translation after merging.

**Review by doulba — APPROVED** [review]

**Pull request merged**